extensions/common/iam/identity-trust/identity-trust-core/src/main/java/org/eclipse/edc/iam/identitytrust/core/IatpDefaultServicesExtension.java

```diff
@@ -87,6 +110,13 @@ public ScopeExtractorRegistry scopeExtractorRegistry() {
         return new IatpScopeExtractorRegistry();
     }
 
+    @Provider(isDefault = true)
+    public CredentialServiceClient createClient(ServiceExtensionContext context) {
+        return new DefaultCredentialServiceClient(httpClient, Json.createBuilderFactory(Map.of()),
+                typeManager.getMapper(JSON_LD), typeTransformerRegistry, jsonLd, context.getMonitor(),
+                context.getConfig().getString(CREDENTIALSERVICE_URL_PROPERTY));
```

Comment:
@paullatzelsperger are we sure `jsonLd` will be injected before this method is invoked? (as it is a `isDefault = true`)